### PR TITLE
bugfix: search UI groups and server recommendations

### PR DESCRIPTION
**Pull request by skeptrunedev**

### CODE REVIEW

File: search/src/components/Atoms/AfMessage.tsx [hunk 1]
```diff
@@ -29,32 +28,7 @@ export const AfMessage = (props: AfMessageProps) => {
...
-    }
-    chunkNumList.sort((a, b) => a - b);
-    for (const num of chunkNumList) {
-      const chunk = props.chunks()[num - 1];
-      chunk.score = num;
-      if (!metadata().includes(chunk)) {
-        // the linter does not understand that the chunk can sometimes be undefined or null
-        // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
-        if (!chunk) return;
-        setMetadata((prev) => [...prev, chunk]);
```
Comment (densumesh) [inline search/src/components/Atoms/AfMessage.tsx:55]: Are we not doing the citations for the chat anymore?

Comment (skeptrunedev) [reply search/src/components/Atoms/AfMessage.tsx:55]: We are, but I think it's just better to show all the selected cards all the time. Trying to select only the used ones was too inconsistent.

**Review by densumesh — APPROVED** [review]

File: server/src/handlers/message_handler.rs [hunk 2]
```diff
@@ -895,9 +895,9 @@ pub async fn create_suggested_queries_handler(
         ServerDatasetConfiguration::from_json(dataset_org_plan_sub.dataset.server_configuration);
     let base_url = dataset_config
         .LLM_BASE_URL
-        .unwrap_or("https://openrouter.ai/api/v1".into());
+        .unwrap_or("https://api.openai.com/api/v1".into());
```
Comment (fastfists) [inline server/src/handlers/message_handler.rs:898]: Wait are we defaulting to openrouter or open ai. The model names seem inconsistent.

File: server/src/handlers/chunk_handler.rs [hunk 3]
```diff
@@ -1106,7 +1106,7 @@ pub async fn generate_off_chunks(
     });
 
     let parameters = ChatCompletionParameters {
-        model: data.model.clone().unwrap_or("openai/gpt-3.5-turbo-1106".to_string()),
+        model: data.model.clone().unwrap_or("mistralai/mixtral-8x7b-instruct".to_string()),
```
Comment (fastfists) [inline server/src/handlers/chunk_handler.rs:1109]: Mixtral used here

**Pull request merged**